README.md

```diff
@@ -2,6 +2,12 @@
 
 Another Kubernetes Helper
 
+## Dependencies
+Some of the bootstrap scripts depend on Python 2 and pyyaml being installed
+```sh
+pip install pyyaml
```

Comment:
We can actually omit this since the usage of scripts is purely up to the user. They'll know what deps they need to run their scripts.

Reply:
Ah good point that should go in resources_ankh if anything